src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplate.tsx

```diff
@@ -0,0 +1,106 @@
+import { useFieldArray } from 'react-hook-form';
+import { Box, Button, IconButton } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useMemo } from 'react';
+
+import {
+  StyledFlexColumn,
+  StyledFlexTopCenter,
+  StyledLabelBoldLarge,
+  StyledTitleMedium,
+  variables,
+} from 'shared/styles';
+import { Svg, Tooltip } from 'shared/components';
+import { InputController } from 'shared/components/FormComponents';
+import { useCustomFormContext } from 'modules/Builder/hooks';
+import { Item } from 'redux/modules';
+
+import { ItemOptionContainer } from '../ItemOptionContainer';
+import { PhrasalTemplatePhrase } from './PhrasalTemplatePhrase';
+import { StyledPhrasalTemplateList } from './PhrasalTemplate.styles';
+import { getActivityItemsForResponseField, getNewDefaultPhrase } from './PhrasalTemplate.utils';
+
+export const PhrasalTemplate = ({ name = '' }: { name?: string }) => {
+  const [activitiesIndexString, itemIndexString] = name.split('.items.');
+  const { t } = useTranslation('app');
+  const phraseName = `${name}.responseValues.phrases`;
+  const { control, getValues, register } = useCustomFormContext();
+  const { fields, append, remove } = useFieldArray({
+    control,
+    name: phraseName,
+    rules: { minLength: 1 },
+  });
+  const { items = [] }: { items?: Item[] } = getValues(activitiesIndexString) ?? {};
+  const currentItemIndex = parseInt(itemIndexString, 10);
+  const responseOptions = useMemo(
+    () => getActivityItemsForResponseField(items, currentItemIndex),
+    [currentItemIndex, items],
+  );
+
+  const handleAddPhrase = () => {
+    append(getNewDefaultPhrase());
+  };
+
+  return (
+    <StyledFlexColumn sx={{ gap: 2.4, placeContent: 'flex-start' }}>
+      <StyledTitleMedium>{t('phrasalTemplateItem.lede')}</StyledTitleMedium>
+
+      <ItemOptionContainer sx={{ mb: 0, pb: 0 }} title="">
```

Comment:
Thanks for making `title` optional :pray: `title` prop can now be removed:
```suggestion
      <ItemOptionContainer sx={{ mb: 0, pb: 0 }}">
```